garak/analyze/aggregate_reports.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python3
+
```

Comment:
These additions in utility classes may not provide the utility intended. Happy to leave them for now.

Would you ever expect a user to execute:
```
./aggregate_reports.py [args]
```

Reply:
yes, definitely!